priv/sticker_options/en/contacts.txt

```diff
@@ -0,0 +1 @@
+WhatsApp
```

Comment:
should probably be deleted

Reply:
Since it's not one of the known stickers (https://backend2.getsince.app/admin/stickers), it's not used anyway.